test/run

```diff
@@ -48,6 +53,11 @@ def run_cmd(cmd):
         sys.exit(1)
 
 
+def get_test_module(module_file):
+    sys.path.insert(1, str(module_file.parent))
```

Comment:
I propose you change this function to:

```
def is_slow_test(module_file):
    try:
        return importlib.import_module(str(module_file.stem)).SLOW
    except AttributeError:
        logging.error("Variable SLOW is missing from: %s", module_file.name)
        sys.exit(1)
```

Reply:
Currently, you are inserting `str(module_file.parent))` for each `module_file`.

This results into `/my-custom-path/aws-build-accumulator/test/e2e/tests` being added 21 times (the total number of tests minus 1) being added to `sys.path`. It should be added only once.